src/InputManager.cpp

```diff
@@ -39,4 +48,33 @@ void InputManager::KeyInput()
 	{
 		glfwSetWindowShouldClose(window, GL_TRUE);
 	}
+	else if(glfwGetKey(window, GLFW_KEY_UP) == GLFW_PRESS)
+	{
+		camera->Zoom(true);
+	}
+	else if(glfwGetKey(window, GLFW_KEY_DOWN) == GLFW_PRESS)
+	{
+		camera->Zoom(false);
+	}
+	else if(glfwGetKey(window,GLFW_KEY_LEFT_SHIFT) == GLFW_PRESS)
+	{
+		shiftKey = true;
+	}
+	else if(glfwGetKey(window,GLFW_KEY_LEFT_SHIFT) == GLFW_RELEASE)
+	{
+		shiftKey = false;
+	}
+
 }
+
+void InputManager::ScrollWheel(double xoffset, double yoffset){
+    if(yoffset == 1 ){
+			camera->Zoom(true);
+		}
+    else if (yoffset == -1){
```

Comment:
Left bracket needs to go to next line to keep consistent with the rest of the code.

Reply:
d